### PR TITLE
Fix: Altered API Data

### PR DESCRIPTION
**Pull request by krak3n**
The API data has altered. Now returns `uri` instead of `spotify_uri`.

### CODE REVIEW

File: tests/test_slack.py [hunk 1]
```diff
@@ -32,17 +32,17 @@ class TestQueryApi(BaseTestCase):
 
     def test_tracks_request(self):
 
-        get = mock.MagicMock(status_code=200, data={'name':'some name'})
+        get = mock.MagicMock(status_code=200, data={'name': 'some name'})
         self.requests.get.return_value = get
-        response = query_api('http://api.thisissoon.fm', 'spotify_uri')
```
Comment (krak3n) [inline tests/test_slack.py:37]: Remember - Don't create variables that you won't use, waste of memory.

File: tests/test_slack.py [hunk 2]
```diff
@@ -75,7 +75,6 @@ def test_slack_post_error(self):
             'image': 'http://albumimage.url'
         }
 
-        response = slack_post('http://slack.com', track['name'], track['artist'], track['album'], track['image'])
+        slack_post('http://slack.com', track['name'], track['artist'], track['album'], track['image'])
 
         self.logger.error.assert_called_once()
-
```
Comment (krak3n) [inline tests/test_slack.py:81]: Never leave blank lines at end of files.

File: tests/test_slack.py [hunk 3]
```diff
@@ -32,17 +32,17 @@ class TestQueryApi(BaseTestCase):
 
     def test_tracks_request(self):
 
-        get = mock.MagicMock(status_code=200, data={'name':'some name'})
+        get = mock.MagicMock(status_code=200, data={'name': 'some name'})
```
Comment (krak3n) [inline tests/test_slack.py:35]: Always should have a space right side of `:` in dictionaries.

**Comment by jamesjwarren** [comment]
Thanks :+1:

**Pull request merged**